ui/pages/confirm-transaction-base/confirm-transaction-base.component.js

```diff
@@ -488,48 +484,44 @@ export default class ConfirmTransactionBase extends Component {
           }
           detailTitleColor={COLORS.BLACK}
           detailText={
-            !isMultiLayerFeeNetwork && (
-              <div className="confirm-page-container-content__currency-container">
-                {renderHeartBeatIfNotInTest()}
-                <UserPreferencedCurrencyDisplay
-                  type={SECONDARY}
-                  value={hexMinimumTransactionFee}
-                  hideLabel={Boolean(useNativeCurrencyAsPrimaryCurrency)}
-                />
-              </div>
-            )
+            <div className="confirm-page-container-content__currency-container test">
+              {renderHeartBeatIfNotInTest()}
+              <UserPreferencedCurrencyDisplay
+                type={SECONDARY}
+                value={hexMinimumTransactionFee}
+                hideLabel={Boolean(useNativeCurrencyAsPrimaryCurrency)}
+              />
+            </div>
           }
           detailTotal={
             <div className="confirm-page-container-content__currency-container">
               {renderHeartBeatIfNotInTest()}
               <UserPreferencedCurrencyDisplay
                 type={PRIMARY}
                 value={hexMinimumTransactionFee}
-                hideLabel={!useNativeCurrencyAsPrimaryCurrency}
-                numberOfDecimals={isMultiLayerFeeNetwork ? 18 : 6}
+                hideLabel={false && !useNativeCurrencyAsPrimaryCurrency}
+                numberOfDecimals={6}
               />
             </div>
           }
           subText={
-            !isMultiLayerFeeNetwork && (
```

Comment:
No code changes here except the removal of `!isMultiLayerFeeNetwork && ` and then all the code having to change indentation